src/js/components/award/details/DetailsSection.jsx

```diff
@@ -97,12 +98,37 @@ export default class DetailsSection extends React.Component {
     render() {
         const content = this.currentSection();
 
+        const tabs = [
```

Comment:
Is there a reason this is in the `render` function instead of outside the class?